### PR TITLE
Search hotfix release (11.2.2)

### PR DESCRIPTION
**Pull request by kashifkhan**
Fix for a user reported issue around not being able to assign a Cognitive Svc Key to a Skillset.

3 Models - `CognitiveServicesAccount`, `DefaultCognitiveServicesAccount`, `CognitiveServicesAccountKey` were exposed that let a user programmatically assign keys etc to the underlying cognitive service skill

### CODE REVIEW

**Review by tjprescott — CHANGES_REQUESTED** [review]
You need a CHANGELOG entry and version bump to release this.

File: sdk/search/azure-search-documents/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 11.2.2 (2022-04-14)
+
+**Fixes**
+* Fixes a bug allowing users to set keys for cognitive service skills using the API. Exposes `DefaultCognitiveServicesAccount` and `CognitiveServicesAccountKey`
```
Comment (tjprescott) [inline sdk/search/azure-search-documents/CHANGELOG.md:6]: Is this the right syntax?  I think our auto-release notes relies on have the exact right format for these headers.

Comment (kashifkhan) [reply sdk/search/azure-search-documents/CHANGELOG.md]: I've seen it multiple ways, but got it consistent :)

File: sdk/search/azure-search-documents/CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 ## 11.2.2 (2022-04-14)
 
-**Fixes**
-* Fixes a bug allowing users to set keys for cognitive service skills using the API. Exposes `DefaultCognitiveServicesAccount` and `CognitiveServicesAccountKey`
+### Fixes
```
Comment (tjprescott) [inline sdk/search/azure-search-documents/CHANGELOG.md]:
```suggestion
### Bugs Fixed
```

**Review by tjprescott — CHANGES_REQUESTED** [review]

**Review by tjprescott — APPROVED** [review]

**Pull request merged**